package.json

```diff
@@ -42,8 +42,10 @@
     "react-dom": "15.6.1",
     "react-helmet": "5.1.3",
     "react-redux": "5.0.5",
-    "redux": "3.7.2",
+    "redux": "3.7.1",
+    "redux-form": "^7.0.0",
```

Comment:
I thought it made sense when using redux, especially to not deal with too much React states for validation, what do you decide?